reliability/single-fit/trace/src/test/kotlin/com/datadog/android/trace/integration/otel/OtelTracerProviderTest.kt

```diff
@@ -803,7 +803,7 @@ internal class OtelTracerProviderTest {
         val keptSpans = spansWritten.filter {
             it.getInt(SAMPLING_PRIORITY_KEY) == PrioritySampling.USER_KEEP.toInt()
         }
-        val offset = 10
+        val offset = 20
```

Comment:
I think the test will be still flaky, even with this change. And 20% is quite big offset, maybe it makes sense only a small absolute values. And if you have actual say 1, and expected as 2, this won't help (because diff will be 50% for such numbers).

Reply:
I tested with 1000 repeated tests and never failed. I think it is fine but also my question is regarding the offset as it is too high. Should we still keep this test ?

Reply:
I don't have a definitive answer here. I think we have similar tests checking sampling rate in other modules, and they probably don't have a high offset? Maybe we can modify the test somehow?

Reply:
yes...will try to have a look what else I can do

Reply:
> [!NOTE]
> One way to limit flakyness is to increase the `numberOfSpans`